addons/website_slides_survey/models/slide_channel.py

```diff
@@ -1,10 +1,49 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class ChannelUsersRelation(models.Model):
+    _inherit = 'slide.channel.partner'
+
+    certification_passed = fields.Boolean("Certified", compute="_compute_certification_passed", store=True)
```

Comment:
picky std: for consistency with fields definition in the same file
```suggestion
    certification_passed = fields.Boolean("Certified", compute='_compute_certification_passed', store=True)